instrumentation/opentelemetry-instrumentation-sklearn/tests/test_sklearn.py

```diff
@@ -0,0 +1,126 @@
+from sklearn.ensemble import RandomForestClassifier
+
+from opentelemetry.instrumentation.sklearn import (
+    DEFAULT_EXCLUDE_CLASSES,
+    DEFAULT_METHODS,
+    SklearnInstrumentor,
+    get_base_estimators,
+)
+from opentelemetry.test.test_base import TestBase
+from opentelemetry.trace import SpanKind
+
+from .fixtures import pipeline, random_input
+
+
+class TestSklearn(TestBase):
+    def test_package_instrumentation(self):
+        ski = SklearnInstrumentor(packages=["sklearn"])
```

Comment:
Is it necessary to pass `["sklearn"]` to the constructor here? I am under the impression that this is done by default because of [this](https://github.com/open-telemetry/opentelemetry-python/pull/1054/files#diff-d7d417605a6f41fa00cbe80a2629ab85R32).

Reply:
It's not necessary, no. I can remove it.